src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -69,6 +69,8 @@ export class BrowsingContextImpl {
   };
 
   #url = 'about:blank';
+  // Required to provide navigation info in case of navigation was canceled.
+  #ongoingNavigationUrl?: string;
```

Comment:
This URL doesn't seem to be used for anything.

AI:

- either make this a boolean, OR
- print it alongside "navigation canceled", for more useful logging

Reply:
You are right. At first I was going to implement `browsingContext.navigationAborted` event. But then decided to leave it aside, as just failing navigation would be enough for Puppeteer.